### DISCUSSION TITLE
Document running Dagger on a Jenkins cluster

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#12516] [Ideas]

**Comment by nipuna-perera** [comment]
Hey @kpenfound, we run Cloudbees Core with ephemeral agents. I'd be happy to provide details about our setup. I haven't gotten dagger to run on it yet. Not because it's not possible but because I haven't had the time. However, not having a local cache isn't encouraging me much to get on it.

**Comment by mircubed** [comment]
@rawkode This might be a good one to build a real-world example for in the video series
